macos/FluentUI/Button/Button.swift

```diff
@@ -587,6 +602,7 @@ private struct ButtonSizeParameters {
 	fileprivate let titleVerticalPositionAdjustment: CGFloat
 	fileprivate let titleToImageSpacing: CGFloat
 	fileprivate let titleToImageVerticalSpacingAdjustment: CGFloat
+	fileprivate var minButtonHeight: CGFloat?
```

Comment:
might as well not make it options. based on the comment of the code, large min height is 28 and small is 20?

Reply:
One thing to call out. I've adjusted the min button heights as 28 and 20 for large and small respectively. However I noticed that for retina screen, the small button is 38 pts high (so amounting to 19 instead of 20) while for non-retina it would be as expected viz.20

Now with the minButtonHeight added as 20, it will be 20 in both cases.